utensor_cgen/backend/transformer.py

```diff
@@ -0,0 +1,74 @@
+import pickle
+
+from utensor_cgen.logger import logger
+from utensor_cgen.transformer.pipeline import TransformerPipeline
+from utensor_cgen.utils import class_property
+
+from .base import BackendPart
+
+__all__ = ['PipelineTransformer']
+
+class PipelineTransformer(BackendPart):
+  TARGET = 'generic'
+  PART = 'pipeline_transformer'
+
+  def __init__(self, config):
+    self.transformer = TransformerPipeline(
+      methods=self.config['transform_methods']
+    )
+    self.trans_methods = self.config['transform_methods']
+    self.save_graph = self.config['save_graph']
+
+  def transform(self, ugraph):
+    logger.info("Transforming graph: %s", ugraph.name)
+    logger.info("Transform pipeline: %s", ' -> '.join(self.trans_methods))
+    self._check_non_quantized(ugraph)
```

Comment:
scandalous

Reply:
I think it's better to make sure users pass a non quantized graph to the transformer.
If quantization is required, it's better to be done by the transformer backend part.
Not sure if you have other issue in mind.

Reply:
We wont be able to guarantee this with quantized aware training. So it's probably best if we try to bake these checks in now.

Reply:
How about we make exceptions to quantized aware trained graph?
I think there should be some unique ops used exclusive to quantization aware trained graph, right?
Or we can add special attribute value to quantization aware graph.

Reply:
Yeah right now my concern is for the various quantization frameworks. TF and PyTorch for example either make it part of training and then rewrite the graph, or do offline quantization and pass in representative inputs to compute the output ranges of the various operator nodes. Supporting all these runtimes might be a pain.

Reply:
I have no good idea for now.
It's better to work with an example.
Current workflow expecting a non-quantized graph since many transformers are designed for non-quantized graph.
I think it's better to leave it as is for next release and see what is the better way to do it then.
Just for now.

Reply:
@neil-tan thoughts? I can see reasons for both ways for now